<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonEdit</class>
 <widget class="QDialog" name="PersonEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Person</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dobEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dodEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="aliveButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Alive</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">livingGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="deadButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Dead</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">livingGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="maleButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">genderGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="femaleButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">genderGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>68</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDOB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of birth:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of death:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PersonEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PersonEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="genderGroup"/>
  <buttongroup name="livingGroup"/>
 </buttongroups>
</ui>
